### PR TITLE
Fix Typing Error for Padding with asymmetric signatures

### LINKED ISSUE #24806: Typing Error for ConstantPad
## 🐛 Bug

There is a Typing Error for Argument 1 of ConstantPad classes when using different paddings for different sides.

## To Reproduce

```python
import torch.nn as nn

if __name__ == "__main__":
    t1 = nn.ConstantPad1d((3, 1), 3.5)
    t2 = nn.ConstantPad2d((3, 0, 2, 1), 3.5)
    t3 = nn.ConstantPad3d((3, 3, 6, 6, 0, 1), 3.5)
```

```
>>> pipenv run mypy test.py

test.py:4: error: Argument 1 to "ConstantPad1d" has incompatible type "Tuple[int, int]"; expected "Union[int, Tuple[int]]"
test.py:5: error: Argument 1 to "ConstantPad2d" has incompatible type "Tuple[int, int, int, int]"; expected "Union[int, Tuple[int, int]]"
test.py:6: error: Argument 1 to "ConstantPad3d" has incompatible type "Tuple[int, int, int, int, int, int]"; expected "Union[int, Tuple[int, int, int]]"
```

## Expected behavior

This shall not raise mypy error

## Environment

```
PyTorch version: 1.2.0
Is debug build: No
CUDA used to build PyTorch: 10.0.130

OS: Ubuntu 18.04.2 LTS
GCC version: (Ubuntu 7.4.0-1ubuntu1~18.04.1) 7.4.0
CMake version: version 3.10.2

Python version: 3.7
Is CUDA available: Yes
CUDA runtime version: 10.0.130
GPU models and configuration:
GPU 0: GeForce RTX 2080 Ti
GPU 1: GeForce RTX 2080 Ti

Nvidia driver version: 418.67
cuDNN version: /usr/lib/x86_64-linux-gnu/libcudnn.so.7.6.1

Versions of relevant libraries:
[pip3] numpy==1.17.0
[pip3] torch==1.2.0
[conda] Could not collect

Pytorch installed in Pipenv Environment
mypy==0.720
```

## Additional context

None

cc @ezyang

### PR DESCRIPTION
**Pull request by Microsheep**
This PR resolves #24806

### CODE REVIEW

**Comment by Microsheep** [comment]
@ezyang I think this shall fix the typing error.
I also fixed the other types of padding that share the same problem.

**Review by ezyang — APPROVED** [review]
